package.json

```diff
@@ -138,6 +139,7 @@
     "jest-playwright-preset": "^1.7.0",
     "jest-process-manager": "^0.3.1",
     "multihashing-async": "^1.0.0",
+    "node-pre-gyp": "^0.17.0",
```

Comment:
node-pre-gyp is deprecated -- that's why `@mapbox/node-pre-gyp` was added -- but this repo wouldn't build on github-actions without it. not sure if you have a known fix for this.

Reply:
The proper fix is to update all dependencies so `node-pre-gyp` is no longer pulled-in. Unfortunately, for that we need to update js-ipfs dependencies to the latest version: https://github.com/ipfs/ipfs-webui/pull/1840

Using `@mapbox/node-pre-gyp` here sounds like a good band-aid until that happens :+1: